skops/hub_utils/tests/test_hf_hub.py

```diff
@@ -310,3 +311,26 @@ def test_get_column_names():
     expected_columns = [f"column{x}" for x in range(10)]
     X_df = pd.DataFrame(X_array, columns=expected_columns)
     assert _get_column_names(X_df) == expected_columns
+
+
+@pytest.fixture
+def pandas_not_installed():
+    # patch import so that it raises an ImportError when trying to import
+    # pandas. This works because pandas is only imported lazily.
+    def mock_import(name, *args, **kwargs):
+        if name == "pandas":
+            raise ImportError
+        return __import__(name, *args, **kwargs)
+
+    with patch("builtins.__import__", side_effect=mock_import):
+        yield
```

Comment:
Aren't you using the same in the card tests? We could put this in a test utils kinda place and share between tests?

Reply:
Yes, I thought a bit of duplication is okay, but I now moved the fixture to `conftest.py`.